oss_src/process/process_win.cpp

```diff
@@ -39,32 +39,50 @@ bool process::launch(const std::string &cmd,
   STARTUPINFO startup_info;
   ZeroMemory(&proc_info, sizeof(PROCESS_INFORMATION));
   ZeroMemory(&startup_info, sizeof(STARTUPINFO));
+  startup_info.cb = sizeof(startup_info);
 
   char *c_arglist = convert_args(cmd, args);
 
+  logstream(LOG_INFO) << "Launching process using command: >>> " << c_arglist << " <<< " << std::endl;
+
+  // Set up the proper handlers.
+  startup_info.hStdError = GetStdHandle(STD_ERROR_HANDLE);
+  startup_info.dwFlags |= STARTF_USESTDHANDLES;
+
   BOOL ret;
+
+  ret = DuplicateHandle( GetCurrentProcess(), GetStdHandle(STD_ERROR_HANDLE),
+                         GetCurrentProcess(), &m_stderr_handle, 0, TRUE, DUPLICATE_SAME_ACCESS);
+
+  if(!ret) {
+    auto err = GetLastError();
+    logstream(LOG_WARNING) << "Failed to create duplicate stderr file handle: " << get_last_err_str(err) << std::endl;
+    logstream(LOG_WARNING) << "  Attempting to continue with : " << get_last_err_str(err) << std::endl;
```

Comment:
Is this the intended error message? It seems like it will return the same error string twice.